### ISSUE TITLE
barWidth breaks wavform display

### ISSUE THREAD

**Issue by RighteousIdealizedDung** [katspaugh/wavesurfer.js#907]
Adding barWidth: 3 to the Wavesurfer options removes the waveform entirely. I have created a demo at http://codepen.io/RighteousIdealizedDung/pen/BQXeod. Without the barWidth option, all appears correctly. My code is based on code from http://stackoverflow.com/questions/38918909/how-can-i-load-only-waveform-and-wait-to-user-click-play-to-download-the-audio. I am using server-generated JSON peak data. I have experimented with various combinations of options with no success. It looks like a wavesurfer bug, any suggestions as to how this can be fixed? Thanks

**Comment by katspaugh** [comment]
Yes, definitely a bug in combination of `normalize` and `barWidth`.

**Comment by RighteousIdealizedDung** [comment]
Thanks for the work-around!

**Comment by chrisparton1991** [comment]
Hi guys, I've raised #916 to fix this. The `absmax` calculation in `drawBars()` wasn't taking negative peaks into account, so in this case the max was 0. I simply used the code in `drawWave()`, which does take negatives into account.

We'll need to fix this in the `next` branch too I assume. I don't have time right now but I'll try to get a PR for that in sometime soon. Of course, it's a simple change so anyone can do it :)
